README.md

```diff
@@ -138,7 +138,9 @@ chmod +x cloud-sql-proxy
 
 ```sh
 # see Releases for other versions
-wget https://storage.googleapis.com/cloud-sql-connectors/cloud-sql-proxy/v2.0.0-preview.3/cloud-sql-proxy-x64.exe -O cloud-sql-proxy.exe
+URL="https://storage.googleapis.com/cloud-sql-connectors/cloud-sql-proxy/v2.0.0-preview.3"
```

Comment:
Doesn't windows use set `VAR=VAL` for env variables?

Reply:
@balys I think this needs to be:

```
set URL="https://storage.googleapis.com/cloud-sql-connectors/cloud-sql-proxy/v2.0.0-preview.3"
wget %URL%/cloud-sql-proxy.x64.exe" -O cloud-sql-proxy.exe
```